examples/plot_simulate_evoked.py

```diff
@@ -39,15 +39,17 @@
 
 ###############################################################################
 # and then plot it
-import matplotlib.pyplot as plt
-fig, axes = plt.subplots(2, 1, sharex=True, figsize=(6, 6))
-for dpl in dpls:
-    dpl.plot(ax=axes[0], layer='agg')
-net.plot_input(ax=axes[1])
+if get_rank() == 0:
```

Comment:
why not put this logic inside the plotting function?

Reply:
It would be harmless to include in the plotting function as well to keep examples simple. However, what if the user has their own code that they don't want to run on all cores. Plotting is just one example that should be done on a single core. For example, a user might perform additional analysis on data in the Dipole object (maybe RMSE or something else of their choosing). I believe this functionality should still be exposed.

Reply:
That's fair, but could you:

1) add some documentation in the docstring of the example explaining how you would run it with MPI
2) add `get_rank` to api.rst with a docstring so it's a public function

Reply:
`get_rank` is back to not being exposed for now.